### PR TITLE
Fixes #21238 - Settings - show defaults on hover

### PR DESCRIPTION
**Pull request by swapab**
Defaults visible only when default and actual value are different

### CODE REVIEW

**Review by dLobatog — CHANGES_REQUESTED** [review]
@swapab I think as an user I would be less confused if I saw "default: xxxx" on hover in *all* cases, not only when the current value is something (not the default). Code-wise - 5 :star: :smile:

ccing @rohoover if you could give your opinion

File: test/models/setting_test.rb [hunk 1]
```diff
@@ -600,4 +600,14 @@ def sticky_setting
     refute_empty sorted_list
     assert sorted_list.keys.exclude?(sticky_setting)
   end
+
+  test 'full_name_with_default should include default value' do
+    setting = Setting.create(:name => 'foo', :value => 'baz', :default => 'boo', :description => 'test foo', :full_name => 'Foo Name')
```
Comment (dLobatog) [inline test/models/setting_test.rb]: Can you try `Factory.build` to avoid creating an object in db (slower)?

**Comment by swapab** [comment]
@dLobatog +1
I had thought of the same :smile: Let user's see default settings by default on hover.

File: test/models/setting_test.rb (hunk 1 above)
Comment (swapab) [reply test/models/setting_test.rb]: Agreed. Done.

**Comment by Rohoover** [comment]
I'm good with that.

**Comment by swapab** [comment]
Why didn't the Jenkins triggered a new job after a new `git push`? New Foreman build particularly.

**Comment by tbrisker** [comment]
[test foreman]

**Comment by tbrisker** [comment]
@swapab can you rebase please on latest develop so tests pass?

**Review by dLobatog — APPROVED** [review]

**Review by dLobatog — DISMISSED** [review]
Thanks @swapab , please rebase and push to get tests running again. I'll merge if 🍏

**Comment by swapab** [comment]
@dLobatog rebased and it's :green_apple: !

**Review by tbrisker — APPROVED** [review]

**Comment by tbrisker** [comment]
Thanks @swapab !

**Pull request merged**